MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -12,5 +13,13 @@ internal class MiniAppWebView: WKWebView {
         self.init(frame: .zero, configuration: config)
         contentMode = .scaleToFill
         load(urlRequest)
+        evaluateJavaScript("navigator.userAgent") { [weak self] (result, error) in
+            if error != nil {
+                return
+            }
+            if let userAgent = result as? String {
+                self?.customUserAgent = userAgent + " " + environment.hostAppUserAgentInfo
```

Comment:
Will this set an extra space on the end of the user agent string even if `environment.hostAppUserAgentInfo` is not set? Can we check if `environment.hostAppUserAgentInfo` is empty before running the `evaluateJavascript` above?

I'm not sure if adding an extra space really matters, but it's better to not modify the user agent at all if there's no custom string to append.

Reply:
Sure, will add it 👍